components/LoadingSkeleton/LoadingSkeleton.tsx

```diff
@@ -1,30 +1,47 @@
 import { loadingSkeletonOpacity100, white, whiteOpacity10 } from "constant";
+import { cloneElement } from "react";
 import styled, { CSSProperties, keyframes } from "styled-components";
 
 interface Props {
-  width?: string | number;
-  height?: string | number;
+  children: JSX.Element;
+  isLoading: boolean;
+  height?: CSSProperties["height"];
+  width?: CSSProperties["width"];
   variant?: "grey" | "white";
 }
 export function LoadingSkeleton({
```

Comment:
I reworked this so that you pass the content that it will be placeholder for as children. This lets us get the height of the loader from the child, and we don't have to specify different heights for different breakpoints. We can let the fluid typography handle it for us.